src/style.css

```diff
@@ -1 +1,9 @@
-/* styles go here */
+.form-fieldset {
```

Comment:
Class is called ```form-fieldset``` but it is used to style ```label``` elements.

Please also look at the other class which has a similar problem.